src/core/sys/linux/sys/inotify.d

```diff
@@ -48,6 +48,9 @@ enum: uint
     IN_ALL_EVENTS    = 0x80000FFF,
 }
 
+// Old typo, preserved for compatibility
+enum IN_UMOUNT = IN_UNMOUNT;
```

Comment:
Maybe later. We need at least one release where you can use either an old feature or its replacement without any warnings.